sql-plugin/src/main/java/com/nvidia/spark/rapids/GpuColumnVector.java

```diff
@@ -713,7 +715,10 @@ static boolean typeConversionAllowed(ColumnView cv, DataType colType) {
       }
       return true;
     } else if (colType instanceof BinaryType) {
-      if (!(dt.equals(DType.LIST))) {
+      SQLConf conf = SQLConf.get();
```

Comment:
I am not happy with this change. This code gets called in more places than just when we read in parquet. With this do we have to make sure that all inputs to any binary operation work on both String and List? How do we test those?

Reply:
Hmm.. I didn't think about that. You are right. If a `binaryAsString` is set other ops could fail

The other thing I can think of is to evolve schema to handle this